Examples/react-application-insights-demo/.gitignore

```diff
@@ -1,23 +0,0 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
```

Comment:
You;'ll move eerything under Examples\ to the samples repo right?

Reply:
Yes. only 1 sample was in Examples/. now located [here](https://github.com/Azure-Samples/application-insights-react-demo)